pandas/io/tests/test_parsers.py

```diff
@@ -2415,6 +2415,72 @@ def test_int64_overflow(self):
             expected = pd.DataFrame([str(x)])
             tm.assert_frame_equal(result, expected)
 
+    def test_eof_states(self):
+        ## With skip_blank_lines = True
```

Comment:
I put them right below this by the individual tests. I can make them more prominent.